NotesBackEnd/src/Notes.Api/ConnectionStrings.config

```diff
@@ -0,0 +1 @@
+
```

Comment:
And, please, remove this file from your repository - inability to build or red icon in IDE is good indicator new developer that he is missing something, but having an empty file instead of non-empty is a bit hard to spot…

Reply:
I removed this file but Appveyor build stop working so I added before_build/after_build script to appveyor.yml which creates/deletes ConnectionStrings.config file for build purposes.